projects/step-frontend/src/lib/modules/timeseries/modules/_common/components/time-range-picker/time-range-picker.component.ts

```diff
@@ -27,7 +27,7 @@ import { TimeRange } from '@exense/step-core';
   templateUrl: './time-range-picker.component.html',
   styleUrls: ['./time-range-picker.component.scss'],
   standalone: true,
-  imports: [COMMON_IMPORTS],
+  imports: [COMMON_IMPORTS, MatTooltip],
```

Comment:
MatTooltip will be included with `StepCore` which is a part of `COMMON_IMPORTS`
```suggestion
  imports: [COMMON_IMPORTS],
```